website/docs/docs/build/sl-getting-started.md

```diff
@@ -14,38 +14,34 @@ import DefineMetrics from '/snippets/_sl-define-metrics.md';
+This getting started page presents a sample workflow to help you create your first metrics in dbt Cloud or the command line (CLI). It uses the [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template) as the project data source and is available for you to use. If you prefer, you can create semantic models and metrics for your own dbt project.
 
-To fully experience the power of a universal dbt Semantic Layer, take the following steps:
+This guide will cover the following sections and steps:
 
-- [Create a semantic model](#create-a-semantic-model) with MetricFlow
-- [Define metrics](#define-metrics) with MetricFlow
-- [Test and query metrics locally](#test-and-query-metrics) with MetricFlow
+- [Create a semantic model](#create-a-semantic-model) using MetricFlow
+- [Define metrics](#define-metrics) using MetricFlow
+- [Test and query metrics locally](#test-and-query-metrics) using MetricFlow
 - [Run a production job](#run-a-production-job) in dbt Cloud
 - [Set up dbt Semantic Layer](#set-up-dbt-semantic-layer) in dbt Cloud
 - [Connect to and query the API](#connect-and-query-api) with dbt Cloud
 
+To fully experience the power of a universal [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), you'll need a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account. 
+
 ## Prerequisites
 
 - Have an understanding of key concepts in [MetricFlow](/docs/build/about-metricflow), which powers the revamped dbt Semantic Layer.
 - Have both your production and development environments running dbt version 1.6 or higher. Refer to [upgrade in dbt Cloud](/docs/dbt-versions/upgrade-core-in-cloud) for more info.
-- Use MetricFlow with Snowflake, BigQuery, Databricks, Postgres (CLI only), or Redshift. (dbt Cloud Postgres support coming soon)
-- A successful run in the environment where your Semantic Layer is configured
-- To query with dbt Cloud:
-  * Have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America (Other regions coming soon)
-  * Set up the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions.
-- To query locally:
-  * dbt Core or Developer accounts must manually install the [MetricFlow CLI](/docs/build/metricflow-cli) to test or query their metrics. After installing the package, make sure you run at least one model. <br />
-
+- Use Snowflake, BigQuery, Databricks, Postgres (CLI only), or Redshift. (dbt Cloud Postgres support coming soon)
+-  Create a successful run in the environment where you configure the Semantic Layer. 
+   - **Note:** Semantic Layer currently supports the Deployment environment. (_development experience coming soon_)
```

Comment:
Can you clarify what this means? I think it means that it will work in deployment, but we can't query in dev, is that right?